File: backend/src/main/java/rs/ac/uns/ftn/sbz/backend/model/Physician.java
package rs.ac.uns.ftn.sbz.backend.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Set;

@Entity
@Table(name = "physician")
@Getter
@Setter
@NoArgsConstructor
public class Physician extends Person
{
    public Physician(Long id, String firstName, String lastName, String personalId)
    {
        super(id, firstName, lastName, personalId);
    }
}

File: frontend/src/main/frontend/src/app/shared/model/domain/Physician.ts
import {Ingredient} from './Ingredient';

export class Physician
{
  id: number;
  firstName: string;
  lastName: string;
  personalId: string;

  constructor();
  constructor(id: number, firstName: string, lastName: string, personalId: string);

  constructor(id?: number, firstName?: string, lastName?: string, personalId?: string)
  {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.personalId = personalId;
  }
}
